hera_sim/visibilities/fftvis.py

```diff
@@ -0,0 +1,318 @@
+        simulator is that if requesting polarized output and only a subset of the
+        simulated pols are available in the UVdata object, the code will issue a warning
+        but otherwise continue happily, throwing away the simulated pols it can't store
+        in the UVdata object. Conversely, if polarization is not requested and multiple
+        polarizations are present on the UVData object, it will error unless
+        ``allow_empty_pols`` is set to True (in which case it will warn but continue).
+        The "unpolarized" output of ``fftvis`` is expected to be XX polarization, which
+        corresponds to whatever the UVData object considers to be the x-direction
+        (default East).
+    precision : int, optional
+        Which precision level to use for floats and complex numbers.
+        Allowed values:
+        - 1: float32, complex64
+        - 2: float64, complex128
+    use_gpu : bool, optional
+        Whether to use the GPU version of fftvis or not. Default: False.
+    mpi_comm : MPI communicator
+        MPI communicator, for parallelization.
+    ref_time
+        A reference time for computing adjustments to the co-ordinate transforms using
+        astropy. For best fidelity, set this to a mid-point of your observation times.
+        If specified as a string, this must either use the 'isot' format and 'utc'
+        scale, or be one of "mean", "min" or "max". If any of the latter, the value
+        ll be calculated from the input data directly.
+    correct_source_positions
+        Whether to correct the source positions using astropy and the reference time.
+        Default is True if `ref_time` is given otherwise False.
+    check_antenna_conjugation
+        Whether to check the antenna conjugation. Default is True. This is a fairly
+        heavy operation if there are many antennas and/or many times, and can be
+        safely ignored if the data_model was created from a config file.
+    **kwargs
+        Passed through to :class:`~.simulators.VisibilitySimulator`.
+
+    """
+
+    conjugation_convention = "ant1<ant2"
+    time_ordering = "time"
+
+    diffuse_ability = False
+    # __version__ = __version__ # Fill in the version number here
+
+    def __init__(
+        self,
+        precision: int = 1,
+        use_gpu: bool = False,
+        mpi_comm=None,
+        ref_time: str | Time | None = None,
+        correct_source_positions: bool | None = None,
+        check_antenna_conjugation: bool = True,
+        **kwargs,
+    ):
+        assert precision in {1, 2}
+        self._precision = precision
+        if precision == 1:
+            self._real_dtype = np.float32
+            self._complex_dtype = np.complex64
+        else:
+            self._real_dtype = float
+            self._complex_dtype = complex
+
+        if use_gpu:
+            raise ImportError("GPU acceleration is currently not supported for fftvis.")
+
+        self._fftvis = fftvis.simulate.simulate
```

Comment:
This made sense in matvis because we had to switch between cpu/gpu, but here let's just use fftvis.simulate.simulate directly in the `simulate()` method

Reply:
Changed this to just call `fftvis.simulate.simulate`